python/defaults/Moriond18.py

```diff
@@ -7,11 +7,11 @@
 mc = {
 	'inputFiles' : ['/store/mc/RunIIFall17MiniAOD/TTTo2L2Nu_TuneCP5_PSweights_13TeV-powheg-pythia8/MINIAODSIM/94X_mc2017_realistic_v10-v1/60000/002E7FEA-16E0-E711-922D-0242AC130002.root'],
 	'JPCalibration' : 'JPcalib_MC94X_2017_v1',
-	'mcGlobalTag' : '94X_mc2017_realistic_v10',
+	'mcGlobalTag' : '94X_mc2017_realistic_v12',
 	}
 
 data = {
 	'inputFiles' : ['/store/data/Run2017D/JetHT/MINIAOD/17Nov2017-v1/20000/0249B143-8CCC-E711-BA7C-0025905C2CD0.root'],	
-	'JPCalibration' : 'JPcalib_Data94X_2017_v1',
+        'JPCalibration' : 'TrackProbabilityCalibration_PDF3D_v1_offline',
```

Comment:
I am not entirely sure this is the right one to use. @imarches can you confirm?

Reply:
@mverzett  I don't think it is. Where did you get it from @rauser?

Reply:
@imarches I found it on https://cms-conddb.cern.ch/cmsDbBrowser/list/Prod/gts/94X_dataRun2_ReReco_EOY17_v2 . It's the tag for the record BTagTrackProbability3DRcd. For mc, the JPCalibtration tag (JPcalib_MC94X_2017_v1) was given under that record, so I assumed that I have to take the same record for data. Is this not correct?